kube/examples/proxy.rs

```diff
@@ -0,0 +1,31 @@
+#[macro_use] extern crate log;
+use k8s_openapi::api::core::v1::Namespace;
+
+use kube::{api::{Api, ListParams}, Client, Config};
+use kube::config::KubeConfigOptions;
+
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
+    std::env::set_var("RUST_LOG", "info,kube=debug");
+    env_logger::init();
+
+    let proxy_url = std::env::var("http_proxy").ok();
```

Comment:
I would stick to upper case for an evar

Reply:
My thinking was that `http_proxy` is a defacto standard inspired by curl. `kubectl` too respects this env var.

Reply:
oh really? sorry, i was not aware of that. i'll change it back then.

Reply:
I didn't mean to push for my option.

The example you made is more focused on the important part. Also reqwest lets setting socks5 proxy too with that code, so `PROXY_URL` is strictly speaking a more meaningful name. 

I wrote my comment to just explain why lowercase, and it was before I saw your example.

Reply:
ah, yes, that is true as well - even though the example does not support multiplexing to other reqwest::proxy constructors.

it's nice to be able to match conventions where they exist - so appreciate the reasoning there - but perhaps it's actually better to have the more generic name in this case.

no worries! i'll just leave it as is now then :-)